assets/js/blocks/cart-checkout/mini-cart/component-frontend.tsx

```diff
@@ -31,6 +31,8 @@ const renderMiniCartFrontend = () => {
 		getProps: ( el: HTMLElement ) => ( {
 			isDataOutdated: el.dataset.isDataOutdated,
 			isInitiallyOpen: el.dataset.isInitiallyOpen === 'true',
+			backgroundColor: el.dataset.backgroundColor,
```

Comment:
Taking another look at this... I wonder if we could read the class names from the elements instead of passing the colors in data attributes. I'm thinking something along these lines:

```JS
quantityBadgeClassnames: el.querySelector('.wc-block-mini-cart__quantity-badge').classList
```
Then, we would just pass that array to the React component so it can apply those classes. This way, we wouldn't need to call `getColorClassName()` from the JS code, which allows us to avoid one dependency: `@wordpress/block-editor`. :slightly_smiling_face:

Reply:
Thanks so much, it's a great idea!

Reply:
I update the related components to follow this approach!